examples/openvino/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Intel Corporation
+#
+# Licensed under the BSD License (the "License"); you may not use this file
+# except in compliance with the License. See the license file in the root
+# directory of this source tree for more details.
+
+cmake_minimum_required(VERSION 3.19)
+project(openvino_runner_example)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# Source root directory for executorch.
+if(NOT EXECUTORCH_ROOT)
+  set(EXECUTORCH_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../..)
+endif()
+
+include(${EXECUTORCH_ROOT}/build/Utils.cmake)
+include(${EXECUTORCH_ROOT}/build/Codegen.cmake)
+
+if(NOT PYTHON_EXECUTABLE)
+  resolve_python_executable()
+endif()
+
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Debug)
+endif()
+
+set(_common_compile_options -Wno-deprecated-declarations -fPIC)
+set(_common_include_directories ${EXECUTORCH_ROOT}/..)
+
+set(_openvino_executor_runner__srcs 
+    ${CMAKE_CURRENT_LIST_DIR}/../openvino/executor_runner/openvino_executor_runner.cpp
+)
+
+find_package(executorch CONFIG REQUIRED)
+include_directories(${EXECUTORCH_INCLUDE_DIRS})
+
+# Portable Ops Library
+gen_selected_ops(LIB_NAME "openvino_portable_ops_lib" INCLUDE_ALL_OPS "ON")
+generate_bindings_for_kernels(
+  LIB_NAME "openvino_portable_ops_lib" FUNCTIONS_YAML
+  ${EXECUTORCH_ROOT}/backends/openvino/openvino_functions.yaml
+)
```

Comment:
I dont quite follow why you need to build this? or have openvino_functions.yaml in the first place

Reply:
Thank you for the feedback on this. In case of having ops which are not supported by openvino, we want to create partitions with the supported part of the graph and let unsupported ops to fallback to use portable ops.

We were able to achieve this by having this .yaml file in the build process which includes the ops which are not supported in our translation at the moment. In this case, do you suggest another way to build with portable ops?

Thank you.

Reply:
All you should need to do is to not partition these ops and let there be graph breaks for these. Then in the runtime if you include portable operator library that handles these ops, you should be covered

Reply:
We removed using custom executor runner and changed it to build with common executor runner. We do not need this yaml file anymore. Removed with the latest changes. Fallback works as expected.